File: record/record.py
#!/usr/bin/env python3
class Record:
    def __init__(self, record_id):
        self.id = record_id


def main():
    records = []

    for i in range(16777216):
        r = Record(i)
        records.append(r)


if __name__ == "__main__":
    main()

File: record/record.rb
#!/usr/bin/env ruby

class Record
  attr_accessor :id

  def initialize(record_id)
    @id = record_id
  end
end

def main
  records = []

  (0...8_388_608).each do |i|
    r = Record.new(i)
    records << r
  end
end

main if __FILE__ == $0
